packages/editor/src/browser/diff-command.ts

```diff
@@ -0,0 +1,67 @@
+/*
+ * Copyright (C) 2018 TypeFox and others.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+ */
+
+import { injectable, inject } from 'inversify';
+import URI from '@theia/core/lib/common/uri';
+import { SelectionService } from '@theia/core';
+import { OpenerService } from '@theia/core/lib/browser';
+import { MessageService } from '@theia/core/lib/common/message-service';
+import { FileSystem } from '@theia/filesystem/lib/common/filesystem';
+import { UriAwareCommandHandler, UriCommandHandler, WorkspaceCommands } from '@theia/workspace/lib/browser';
```

Comment:
the editor extension does not depend on the workspace, it is not going to work outside of Theia repo

Reply:
I have completely overlooked that, thanks!

Reply:
As you have [recommended](https://github.com/theia-ide/theia/pull/1438#discussion_r173628101), I have moved the `diff-uris` module to the `@theia/core` and implemented the corresponding handler in the `workspace-commands`.